Mo interview - R-Universe (#517)
ropensci/roweb3: content/blog/2023-03-30-r-universe-stars-3-en/index.md

In her work, they use specialized software because of the information they need to process, such as brain images. Athanasia explains that _"at the end of the day all their statistics are in R, and because of that having our R packages in our tool belt when we are working is super important for our staff."_

Athanasia is working on a project called [Life Brain](https://www.lifebrain.uio.no/), funded by the European Union for the Human Life brand and involving 13 European centers. Mo explains that they have developed a lot of research and made publications, and many results have been implemented in R packages. Asked about the project, with passion and great enthusiasm, Mo says: 

> We are trying to investigate how people are aging, what their brain aging is like, what their cognitive aging is like.

## R-Universe

> The R universe actually came at an extremely timely point for us, 

> is how Athanasia explains the possibilities this platform offered them to [show in a simple way all the software they were working on](https://lifebrain.r-universe.dev/). In this research development, there are different types and sources of funding. Even though they have a collective organization in GitHub, they needed a clear way to show all the work that is distributed in different repositories in the personal accounts of the researchers or the accounts of the research centers. That is why Mo assures that in R-Universe 

Reviewer: I added an 's' to the end of "universe" in the quote. I don't think we need to highlight that, do we?

```suggestion
Athanasia explains that this platform offered them a simple way to [show all the software they were working on](https://lifebrain.r-universe.dev/). In this research development, there are different types and sources of funding. Even though they have a collective organization in GitHub, they needed a clear way to show all the work that is distributed in different repositories in the personal accounts of the researchers or the accounts of the research centers. That is why Mo assures that in R-Universe "*we can create a place for all our universes and thus show the packages.*"
```

> 
> > we can create a place for all our universe and thus show the packages.

Reviewer: ```suggestion

```

> In addition, as Athanasia explains, in R-Universe it is very clear who the developers or maintainers are and that means that everyone keeps their corresponding credits and contributions.

Reviewer: ```suggestion
In addition, Athanasia explains that an important part of using the R-Universe is that it is very clear who the developers or maintainers are, which means that everyone keeps their corresponding credits and contributions.
```

> Another interesting use of R-Universe to share work, and Mo showed us is the possibility of accessing the data but also some extra functions, the same functions they used in their analysis. For example, those in the survey they conducted globally on brain health where obtained more than 27 thousand responses and from which they produced different articles and papers (you can see more information about this work in the video below).  

Reviewer: ```suggestion
Another interesting use of the R-Universe for sharing work that Mo showed us, is the possibility of accessing the data as well as some extra functions used in an analysis. For example, in the Lifebrain Global Brain Health Survey they obtained more than 27 thousand responses, produced articles and papers, *and* provide all the data and code in the [gbhs](https://lifebrain.r-universe.dev/gbhs) package (you can see more information about this work in the video below). 
```

Author: Thanks so much for all the added info here @steffilazerte !

> {{< figure src = "brain_plot_3.png" class = "center" width = "600" alt = "Example of a subcortical map plotted with ggseg" caption = "<center><strong>Subcortical map plot with ggseg</strong></center>">}}

Reviewer: ```suggestion
{{< figure src = "brain_plot_3.png" class = "center" width = "600" alt = "A figure showing two line drawing outlines of a brain. On the left is a top-down image, on the right a side-on image. Both show patches of colour highlighting spots in the centre corresponding to different parts of the brain." caption = "<center><strong>Subcortical map plot with ggseg</strong></center>">}}
```

> While we go through all the packages in the R Universe by Athanasia and Life Brain's work, she shows us her greatest pride: [ggseg](https://ggseg.r-universe.dev/), a development to create images of the brain that can be analyzed from different functions. It was precisely with all this work that Mo found in R-Universe the solution to a  series of problems with the size of the data and the installation of the tools needed to work: 

Reviewer: ```suggestion
As we explore the packages in the [Lifebrain R-Universe](https://lifebrain.r-universe.dev/), Athanasia shows us her greatest pride: [ggseg](https://ggseg.r-universe.dev/), software to create images of the brain that can be analyzed with different functions. It was precisely with this work that Mo found in R-Universe the solution to a series of problems with the size of the data and the installation of the tools needed to work: 
```

> When the data packages are too big and cannot be reduced, it is not possible to publish them in CRAN (...) and failures appear when users want to install them in their development version that took us a long time to solve and with non-optimal results.

> In this context is when R Universe presents her with a definitive solution: 

Reviewer: ```suggestion
In this context is when R-Universe presents her with a definitive solution: 
```

> > This is the real solution to be able to have everything we work on the project with a working installation and not waste any more time troubleshooting our users.

Reviewer: ```suggestion
> This is the real solution, to be able to have everything we work on the project with a working installation and not waste any more time troubleshooting our users.
```

> {{< figure src = "brain_plots.png" class = "center" width = "600" alt = "A serie of three examples plots of different brain atlas" caption = "<center><strong>ggseg in action: ploting brain atlas</strong></center>">}}

Reviewer: This is not a big deal, but is it easy/possible to make the middle plot have the same text size as the others? No worries if it's a pain, I'm just being nit-picky 😅 
```suggestion
{{< figure src = "brain_plots.png" class = "center" width = "600" alt = "A series of three examples plots of different brain atlases. Each row shows four brain outlines with patches of different colours." caption = "<center><strong>ggseg in action: plotting brain atlases</strong></center>">}}
```

Author: I will try to give the same font size.  Each plot comes from different sources and has a different resolution.  When I did the patchwork that result in different font sizes, but I agree that doesn't look as nice as could be.

In Athanasia's experience, R-Universe allowed them to publish all the packages in one place, with easy accessibility and the possibility to receive contributions, maintenance and no installation problems for the users. In her own words: 

> > R-Universe is magic for me.

Reviewer: ❤️

> Remember that R-Universe generates binaries for different operating systems for all packages that belong to the universe, with no limit on the package size. One of the platform’s objectives is to make it easier to publish and install R packages.

Reviewer: ```suggestion
Remember that the R-Universe generates binaries for different operating systems for all packages that belong to the universe, with no limit on the package size. One of the platform’s objectives is to make it easier to publish and install R packages.
```

> In the following video, Athanasia explains in more detail the development of their Life Brain project, how they use R to organize and work with information, and how R-Universe allowed them to organize and share all their packages.

Reviewer: ```suggestion
In the following video, Athanasia explains in more detail the development of their Life Brain project, how they use R to organize and work with information, and how the R-Universe allowed them to organize and share all their packages.
```

> 

Reviewer: ```suggestion
## Video of the interview
```

> {{< vimeo id=807731911 title="Meeting R-Universe Stars - Athanasia Monika Mowinckel" >}}

Reviewer: It's a private video right now. I assume that's just because we're still in draft, but noting that just in case it isn't supposed to be!

Author: Yes, it is because the subtitle still needs an edition. Thanks!
